### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by jricks92**
* Adds option for specifying a dedicated swap partition using `SWAPFILE` variable in `openmiko.conf`
* Adds option for specifying which partition and filesystem type is used to mount to `/sdcard` using `SD_PARTITION` and `SD_FILESYSTEM` variables in `openmiko.conf`. This has been tested so far with `vfat` and `ext2` fs types, but other should work if there are filesystem drivers for them..
* Adds option for specifying a custom `videocapture_settings.json` file using the `VIDEO_CAPTURE_SETTINGS` variable in `openmiko.conf`
* Adds option for specifying custom `autonight` parameters from within `openmiko.conf` using the `AUTONIGHT_PARAMS` variable.
* Adds option for disabling LEDs in `openmiko.conf` with variable `DISABLE_LEDS=1`. This is useful for me since I use a camera as a baby monitor and don't want the blinking lights to wake the baby.
* Adds option for not using `AUTH_REALM` for RTSP stream password (I always have trouble with using it, so I wanted an option to disable it)
* Adds option for enabling or disabling audio support via the `ENABLE_AUDIO` variable in `openmiko.conf` (for those that like to live on the edge 🙂)
* Add option for specifying how many video loopback devices there are in an attempt to hopefully reduce memory consumption.
    * This requires setting up to 3 variables in `openmiko.conf`. I've arbitrarily set this for up to 3 loopback devices max for now. But we can add more if needed. The env variables for this are `VIDEO_DEV_X` where x is 1, 2, or 3.
    * I've also added `/etc/videocapture_settings_1_encoder.json` as an alternative `videocapture_settings.json` file with only 1 encoder for RTSP.
* Implement proper restart/reload commands for `S60camera` init script.

### CODE REVIEW

**Comment by jricks92** [comment]
@tachang have you been able to review this?

**Comment by tachang** [comment]
Thank you for your contribution. I haven't had much chance to review things lately but do plan to get around to it. The biggest blocker I have right now is testing things.

Is there anyone else that wants to try building your firmware and giving it a try? Maybe you can attach an image so people can test it out.

I feel like I've reduced the stability a bit with each new version due to new features and bugs.

File: local.mk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Directly use the submodule source code for builds
-INGENIC_VIDEOCAP_OVERRIDE_SRCDIR=/src/submodules/ingenic_videocap
+# INGENIC_VIDEOCAP_OVERRIDE_SRCDIR=/src/submodules/ingenic_videocap
```
Comment (tachang) [inline local.mk:2]: Why do you comment this out here?

File: overlay_minimal/etc/init.d/S60camera [hunk 2]
```diff
@@ -64,19 +63,44 @@ case "$1" in
...
+		else
+			echo "FFMPEG is already running. Skipping."
+		fi
+	else
+		echo "Audio is set to disabled in openmiko.conf. Skipping starting ffmpeg"
+	fi
+
+
+	# Only set the number of v4l2loopback devices that we need. This hopefully will save some memory.
+	DEVICE_COUNT=1 # Default to 1, not sure if this can be 0 or 1.
```
Comment (tachang) [inline overlay_minimal/etc/init.d/S60camera:80]: Thanks for doing this.

File: local.mk (hunk 1 above)
Comment (jricks92) [reply local.mk:2]: I don't remember actually. I can revert this no problem.

**Comment by jricks92** [comment]
No problem about not having time to review. I have a built copy of this branch here: https://github.com/jricks92/openmiko/releases/tag/v0.0.41-prerelease

AFAIK the user MegaHurtz on Slack has been running this build for a few weeks. I haven't seen any issues come up. I've also been running this firmware on 3 of my Wyze Cam V2s for months. So far it seems to work well. Anything specifically you'd like me to test?

**Comment by tachang** [comment]
For some reason the steam didn't end up starting:
```
# /etc/init.d/S60camera start
Adding necessary camera modules
root: Determined image sensor model as jxf23
insmod: can't insert '/driver/sensor_jxf23.ko': File exists
c
c
c
c
c
sh: 1: unknown operand
Audio is set to disabled in openmiko.conf. Skipping starting ffmpeg
Setting up 1 v4l2loopback device(s)
root: Starting v4l2rtspserver
VIDIOC_STREAMOFF: Bad file descriptor
VIDIOC_REQBUFS: Bad file descriptor
ALSA lib pcm.c:2266:(snd_pcm_open_noupdate) Unknown PCM /dev/video0
```

**Comment by tachang** [comment]
I have an older openmiko.conf. Let me try using yours.

**Comment by tachang** [comment]
Came right up with the newer openmiko.conf file you provided.

I think we'll have to write some sort of upgrade note for openmiko.conf for those using older ones.

**Review by tachang — APPROVED** [review]

**Comment by jricks92** [comment]
@tachang Thanks for merging this in! 

> I think we'll have to write some sort of upgrade note for openmiko.conf for those using older ones.

Would it be better to set default values in the init scripts in case someone doesn't have them in their `openmiko.conf`?

**Pull request merged**